### ISSUE TITLE
ng serve blocked by previous server instance

### ISSUE THREAD

**Issue by maximelafarie** [angular/angular#34831]
# 🐞 Bug report

### Command (mark with an `x`)

- [ ] new
- [x] build
- [x] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc

### Is this a regression?

I don't know. I guess no since I'm launching an Angular project with bazel for the first time.

### Description

It's impossible to serve / build a fresh new project.

I already check across the existing topics about that...
- https://github.com/angular/angular-cli/issues/15011
- https://github.com/bazelbuild/bazel-watcher/issues/252 (which isn't enough at all) 

## 🔬 Minimal Reproduction

After creating a new project with:
```bash
ng new angular-bazel --collection=@angular/bazel
```
Try to start / build the project:

```bash
yarn run start
```

## 🔥 Exception or Error

```bash
yarn run v1.21.1
$ ng serve
WARNING: Waiting for server process to terminate (waited 5 seconds, waiting at most 60)
WARNING: Waiting for server process to terminate (waited 10 seconds, waiting at most 60)
Starting local Bazel server and connecting to it...
Command 'serve' not found. Try 'bazel help'.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

## 🌍 Your Environment
```bash
     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/


Angular CLI: 9.0.0-rc.9
Node: 13.6.0
OS: darwin x64

Angular: 9.0.0-rc.9
... animations, bazel, cli, common, compiler, compiler-cli, core
... forms, language-service, platform-browser
... platform-browser-dynamic, router, upgrade
Ivy Workspace: Yes

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.900.0-rc.9
@angular-devkit/build-angular     0.900.0-rc.9
@angular-devkit/build-optimizer   0.900.0-rc.9
@angular-devkit/build-webpack     0.900.0-rc.9
@angular-devkit/core              9.0.0-rc.9
@angular-devkit/schematics        9.0.0-rc.9
@bazel/bazel                      0.23.2
@bazel/bazel-darwin_x64           0.23.2
@bazel/ibazel                     v0.9.1
@bazel/jasmine                    0.26.0
@bazel/karma                      0.26.0
@bazel/typescript                 0.26.0
@ngtools/webpack                  9.0.0-rc.9
@schematics/angular               9.0.0-rc.9
@schematics/update                0.900.0-rc.9
rxjs                              6.5.4
typescript                        3.6.4
webpack                           4.41.2
```

**Comment by kyliau** [comment]
I tried the exact repro steps with `@angular/bazel` v9.0.0-rc.11 but was unable to replicate the failure.

The error message `Waiting for server process to terminate` indicates that there is already a prior instance of the Bazel server running. Did you run any Bazel command before executing `yarn run start`?

Could you please install the latest rc version and try again? I don't think this is the root cause, but it'll help us narrow down our investigation. Thank you!

**Comment by SilverLinings89** [comment]
I tried to reproduce in the latest version. 
```
     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 9.0.0-rc.12
Node: 10.16.3
OS: linux x64

Angular: 9.0.0-rc.14
... animations, bazel, common, compiler, compiler-cli, core
... forms, language-service, platform-browser
... platform-browser-dynamic, router
Ivy Workspace: Yes

Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.900.0-rc.12
@angular-devkit/core         9.0.0-rc.12
@angular-devkit/schematics   9.0.0-rc.12
@angular/cli                 9.0.0-rc.12
@bazel/bazel                 2.0.0
@bazel/hide-bazel-files      1.2.4
@bazel/ibazel                0.10.3
@bazel/karma                 1.2.2
@bazel/protractor            1.2.2
@bazel/rollup                1.2.2
@bazel/terser                1.2.2
@bazel/typescript            1.2.2
@schematics/angular          9.0.0-rc.12
@schematics/update           0.900.0-rc.12
rxjs                         6.5.4
typescript                   3.7.5
```

`ng serve` and `build` as well as `yarn run start` work for me without errors. this is the package.json:
```{
  "name": "angular-bazel",
  "version": "0.0.0",
  "scripts": {
    "ng": "ng",
    "postinstall": "ngcc --properties es2015 browser module main",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "~9.0.0-rc.14",
    "@angular/common": "~9.0.0-rc.14",
    "@angular/compiler": "~9.0.0-rc.14",
    "@angular/core": "~9.0.0-rc.14",
    "@angular/forms": "~9.0.0-rc.14",
    "@angular/platform-browser": "~9.0.0-rc.14",
    "@angular/platform-browser-dynamic": "~9.0.0-rc.14",
    "@angular/router": "~9.0.0-rc.14",
    "core-js": "^2.5.4",
    "rxjs": "~6.5.3",
    "tslib": "^1.9.0",
    "zone.js": "~0.8.26"
  },
  "devDependencies": {
    "@angular/bazel": "~9.0.0-rc.14",
    "@angular/cli": "~9.0.0-rc.12",
    "@angular/compiler-cli": "~9.0.0-rc.14",
    "@angular/language-service": "~9.0.0-rc.14",
    "@bazel/bazel": "2.0.0",
    "@bazel/ibazel": "0.10.3",
    "@bazel/karma": "1.2.2",
    "@bazel/protractor": "1.2.2",
    "@bazel/rollup": "1.2.2",
    "@bazel/terser": "1.2.2",
    "@bazel/typescript": "1.2.2",
    "@types/node": "~8.9.4",
    "@types/jasmine": "~2.8.8",
    "@types/jasminewd2": "~2.0.3",
    "codelyzer": "~4.5.0",
    "history-server": "1.3.1",
    "html-insert-assets": "0.2.0",
    "jasmine-core": "~2.99.1",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "4.4.1",
    "karma-chrome-launcher": "3.1.0",
    "karma-coverage-istanbul-reporter": "~2.0.1",
    "karma-firefox-launcher": "1.2.0",
    "karma-jasmine": "2.0.1",
    "karma-jasmine-html-reporter": "^0.2.2",
    "karma-requirejs": "1.1.0",
    "karma-sourcemap-loader": "0.3.7",
    "protractor": "5.4.2",
    "requirejs": "2.3.6",
    "rollup": "1.27.5",
    "rollup-plugin-commonjs": "10.1.0",
    "rollup-plugin-node-resolve": "5.2.0",
    "terser": "4.4.0",
    "ts-node": "~7.0.0",
    "tslint": "~5.11.0",
    "typescript": "~3.7.5"
  }
}
```

**Comment by kara** [comment]
@maximelafarie Without a repro, we don't have enough info to keep investigating this. I'm closing this issue given lack of actionable data. Please feel free to open a new issue with a repro case whenever you have one.
